packages/nightingale-sunburst/src/nightingale-sunburst.ts

```diff
@@ -0,0 +1,426 @@
+      context.arc(
+        width / 2,
+        height / 2,
+        segment.y0,
+        segment.x1,
+        segment.x0,
+        true
+      );
+      if (
+        segment.data[this["id-attribute"]] ===
+        this.activeSegment?.data?.[this["id-attribute"]]
+      ) {
+        context.lineWidth = 4;
+        context.globalAlpha = 1;
+      }
+      // Show the stroke
+      context.stroke();
+      context.fill();
+      context.lineWidth = tmpLineW;
+      context.globalAlpha = tmpAlpha;
+    }
+  }
+
+  renderLabels(
+    context: CanvasRenderingContext2D,
+    width: number,
+    height: number
+  ) {
+    context.fillStyle = "black";
+    context.font = `${this["font-size"]}px Arial`;
+    context.textBaseline = "middle";
+    context.textAlign = "center";
+    const labelsToDisplay = this.root
+      ?.descendants()
+      .filter(
+        (d) =>
+          d.depth &&
+          d.depth <= this["max-depth"] &&
+          ((d.y0 + d.y1) / 2) * (d.x1 - d.x0) > 10
+      );
+    for (const segment of labelsToDisplay || []) {
+      const angle = (segment.x0 + segment.x1) / 2;
+      const r = (segment.y0 + segment.y1) / 2;
+      context.save();
+      context.translate(width / 2, height / 2);
+      context.rotate(angle);
+      context.translate(r, 0);
+      let shouldRotate = false;
+      let spaceAvailableToDraw = segment.y1 - segment.y0;
+      // If it's more than a 1/4 of the circle so draw it horizontally
+      if (Math.abs(segment.x1 - segment.x0) > Math.PI / 2) {
+        shouldRotate = true;
+      } else {
+        const availableWidth = getDistanceOfPointsInRadians(
+          { angle: segment.x0, radius: segment.y0 },
+          { angle: segment.x1, radius: segment.y0 }
+        );
+        // If there is more space horizontally than verically then rotate
+        if (availableWidth > segment.y1 - segment.y0) {
+          shouldRotate = true;
+          spaceAvailableToDraw = availableWidth;
+        }
+      }
+      if (shouldRotate) {
+        context.rotate(Math.PI / 2);
+        if (angle > 0 && angle < Math.PI) {
+          context.rotate(Math.PI);
+        }
+      }
+      // rotate left side to make it readable
+      else if (angle > Math.PI / 2 && angle < 1.5 * Math.PI) {
+        context.rotate(Math.PI);
+      }
+
+      context.fillText(
+        segment.data[this["name-attribute"]] ||
+          (segment.data.rank ? `[No ${segment.data.rank}]` : "No name"),
+        0,
+        0,
+        segment.data[this["id-attribute"]] ===
+          this.activeSegment?.data?.[this["id-attribute"]]
+          ? undefined
+          : spaceAvailableToDraw - 2
+      );
+      context.restore();
+    }
+  }
+
+  renderActiveSegmentInfo(context: CanvasRenderingContext2D) {
+    if (this.activeSegment) {
+      context.textAlign = "left";
+      context.font = "bold 12px Arial";
+      context.fillText("Name:", 10, 10);
+      context.fillText(this["weight-attribute-label"], 10, 40);
```

Comment:
Add a colon to be consistent with "Name:"

```suggestion
      context.fillText(`${this["weight-attribute-label"]}:`, 10, 40);
```